graphql-schema/proposal.graphql

```diff
@@ -108,7 +108,15 @@ input QueryProposalsInput {
   searchTerm: String
 }
 
+type ProposalVotesDistribution {
+  yes: Float!
+  no: Float!
+  abstain: Float!
+  noWithVeto: Float!
+}
```

Comment:
looks similar to `ProposalTallyResult`? maybe move `outstandingOption` to Proposal and reuse tallyResult type?

Reply:
`ProposalVotesDistribution` refers to distribution of vote counts by a single wallet address across all proposals voted by this wallet address, so have a slightly different type than `ProposalTallyResult`, which refers to the vote count of all voters on a single proposal.